omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/FlowsheetPageController.java

```diff
@@ -104,6 +104,7 @@ public void controller(@RequestParam(value="patientId", required=false) Patient
 
         Map<String, HtmlForm> flowsheetForms = new LinkedHashMap<String, HtmlForm>();
         Map<String, List<Integer>> flowsheetEncounters = new LinkedHashMap<String, List<Integer>>();
+        Map<Integer, String> encounterTypeIds = new LinkedHashMap<Integer, String>();
```

Comment:
I'd rename this to encounterIdToEncounterTypeUuidMap to make it clearer what it is. (If I'm interpreting it right?)